### PR TITLE
fix(cli): fix signer init strategy

### PR DESCRIPTION
**Pull request by xeno097**
### Description

Probably fixes an issue due to a recent pr merge that changes how signers are initialized based on the command to be executed

### Drive-by changes

- Updates the key info message log level from info to debug to avoid the following
image

### Backward compatibility

- Yes

### Testing

- Manual

### CODE REVIEW

File: typescript/cli/src/context/strategies/chain/MultiChainResolver.ts [hunk 1]
```diff
@@ -58,9 +59,9 @@ export class MultiChainResolver implements ChainResolver {
         return this.resolveRelayerChains(argv);
       case ChainSelectionMode.CORE_APPLY:
         return this.resolveCoreApplyChains(argv);
-      case ChainSelectionMode.ORIGIN_DESTINATION:
+      case ChainSelectionMode.DEFAULT:
       default:
-        return this.resolveOriginDestinationChains(argv);
+        return this.resolveRelayerChains(argv);
```
Comment (paulbalaji) [inline typescript/cli/src/context/strategies/chain/MultiChainResolver.ts:64]: do we still need the RELAYER enum if the default is that behaviour?

Comment (xeno097) [reply typescript/cli/src/context/strategies/chain/MultiChainResolver.ts:64]: makes sense going to remove it

File: typescript/cli/src/context/strategies/chain/ChainResolverFactory.ts [hunk 2]
```diff
@@ -11,13 +10,16 @@ import { ChainResolver } from './types.js';
 export class ChainResolverFactory {
   private static strategyMap: Map<CommandType, () => ChainResolver> = new Map([
     [CommandType.WARP_DEPLOY, () => MultiChainResolver.forWarpRouteConfig()],
-    [CommandType.WARP_SEND, () => MultiChainResolver.forOriginDestination()],
+    // Using the forRelayer resolver because warp send allows the user to self relay the tx
+    [CommandType.WARP_SEND, () => MultiChainResolver.forRelayer()],
```
[the remark this answers is not in the document]
Comment (xeno097) [reply typescript/cli/src/context/strategies/chain/ChainResolverFactory.ts:14]: I think in general I would like to nuke the whole duplicity between signer commands array  commands enum but I want to talk with tx-fusion to see if they added the enum for some specific reason and if no then nuke the whole enum and go back to the old default behaviour where all evm chains were initialized with same value provided in the `HYP_KEY` env

File: typescript/cli/src/context/strategies/chain/MultiChainResolver.ts [hunk 3]
```diff
@@ -151,7 +127,29 @@ export class MultiChainResolver implements ChainResolver {
...
+
+    if (argv.destination) {
+      chains.push(argv.destination);
+    }
+
+    if (!argv.chains) {
+      return Array.from(
+        new Set([...chains, ...this.getEvmChains(multiProvider)]),
+      );
+    }
```
Comment (paulbalaji) [inline typescript/cli/src/context/strategies/chain/MultiChainResolver.ts:145]: thank you!!! 🙏

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**